driver/src/order_driver.rs

```diff
@@ -91,6 +97,19 @@ fn update_balances(state: &mut State, orders: &[Order], solution: &Solution) {
     }
 }
 
+fn get_standing_orders_indexes(standing_orders: &Vec<models::StandingOrder>) -> Vec<u32> {
+    let mut standing_order_indexes = Vec::<u32>::with_capacity(models::NUM_RESERVED_ACCOUNTS as usize);
+        for i in 0..models::NUM_RESERVED_ACCOUNTS {
```

Comment:
nit: Would be more efficient prefill an array with 0s and then iterate over the standing orders, inserting a value into the 0 array where necessary?